src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -53,6 +53,14 @@ public function getConfigTreeBuilder()
                     return $v;
                 })
             ->end()
+            ->beforeNormalization()
```

Comment:
Good spot. Unfortunately, in validate I'll need to explicitly provide assets config.